packages/apputils/src/mainareawidget.ts

```diff
@@ -48,6 +48,7 @@ export class MainAreaWidget<T extends Widget = Widget>
     const toolbar = (this._toolbar = options.toolbar || new ReactiveToolbar());
     toolbar.node.setAttribute('role', 'toolbar');
     toolbar.node.setAttribute('aria-label', trans.__('notebook actions'));
+    toolbar.node.setAttribute('aria-hidden', 'true');
```

Comment:
I think I understand the desire to hide this "toolbar" node because when the content node is the launcher or a normal file or any other number of non-notebook content, then this toolbar node is most often empty, and it looks like this:

![literally a 740px wide by 3px tall box with a bottom border](https://github.com/jupyterlab/jupyterlab/assets/317883/c03fbca2-e58d-4d41-8a32-ee8ee41b3b66)

However, it becomes super important when a notebook is loaded. Then it becomes the node into which the notebook toolbar is loaded, and it looks like this:

![a typical application toolbar with icon buttons for commands like save, add new cell, cut, copy, paste, etc.](https://github.com/jupyterlab/jupyterlab/assets/317883/640385cc-bcfc-47cc-999b-ef4658831963)

Therefore, we can't put aria-hidden on this node toolbar because then it gets hidden from screen readers when  you load a notebook, which means you can't navigate to it using caps lock plus arrow keys, which breaks the expected UX.

Reply:
Yes, makes sense I will amend. Is using caps lock + arrow keys standard to navigate between elements for all assistive screen readers? 
I wasn't aware of this functionality and if so would mean I can remove the some of the changes you have commented on in another PR (https://github.com/jupyterlab/lumino/pull/655) since it will work fine with caps lock + arrow keys